server/src/test/scala/controllers/application/DropInsControllerSpec.scala

```diff
@@ -0,0 +1,77 @@
+package controllers.application
+
+import email.GovNotifyEmail
+import module.DRTModule
+import org.scalatestplus.mockito.MockitoSugar
+import org.scalatestplus.play.PlaySpec
+import play.api.test.Helpers._
+import play.api.test._
+
+class DropInsControllerSpec extends PlaySpec with MockitoSugar {
```

Comment:
This is great! It opens the door to us dropping Play and switching to AkkaHttp which would make our dependency upgrades easier to resolve

Reply:
Yeah also in incremental way